core/nullDC.cpp

```diff
@@ -39,35 +39,25 @@ settings_t settings;
 #include <windows.h>
 #endif
 
-int GetFile(char *szFileName, char *szParse=0,u32 flags=0) 
+int GetFile(char *szFileName, char *szParse = 0, u32 flags = 0)
 {
-	cfgLoadStr("config","image",szFileName,"null");
-	if (strcmp(szFileName,"null")==0)
+#if HOST_OS==OS_WINDOWS
+	cfgLoadStr("config", "image", szFileName, "");
```

Comment:
cfgLoadStr should not be within the windows `#if` check as it applies to all platforms